huaweicloud/endpoints.go

```diff
@@ -100,6 +100,11 @@ var allServiceCatalog = map[string]ServiceCatalog{
 		Name:    "functiongraph",
 		Version: "v2",
 	},
+	"swrv2": {
+		Name:             "swr-api",
+		Version:          "v2",
+		WithOutProjectID: true,
+	},
```

Comment:
should add a acc test in endpoints_test.go

Reply:
done